CHANGELOG.md

```diff
@@ -15,6 +15,11 @@
 * `0.2.x` Releases - [0.2.0](#020) | [0.2.1](#021) | [0.2.2](#022)
```

Comment:
missing an:
* `2.0.98` Releases - [2.0.98](#2098)

Reply:
at the top of the changelog

Reply:
We haven't change anything in the `ExposurePlayback` SDK, but I guess with this release it will have a new release too, So do I need to add the release versions in that SDK too ?